packages/quantic/force-app/main/default/lwc/quanticSearchBoxInput/quanticSearchBoxInput.js

```diff
@@ -311,10 +311,8 @@ export default class QuanticSearchBoxInput extends LightningElement {
 
   clearInput() {
     this.sendInputValueChangeEvent('');
+    this.input.value = '';
     this.input.focus();
-    if (this.textarea) {
-      this.adjustTextAreaHeight();
```

Comment:
this is useless since we already call that whenever the input is focused

Reply:
indeed 👍